Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Utilities/HttpRequestUtility.cs

```diff
@@ -107,7 +107,7 @@ public static (IDictionary<string, string>, IDictionary<string, IList<string>>)
         {
             var key = lowerCaseKeyName ? header.Key.ToLower() : header.Key;
             singleValueHeaders[key] = header.Value.Last() ?? "";
-            multiValueHeaders[key] = [.. header.Value];
+            multiValueHeaders[key] = [.. header.Value!];
```

Comment:
I think the null warnings happening in this file were a false positive. They warnings only show up when doing `dotnet pack` not in VS. And the `Value` is a struct so it can't be null.